### ISSUE TITLE
test-4.0.2 fails to build with MingW

### ISSUE THREAD

**Issue by pmconrad** [bitshares/bitshares-core#2065]
**Build Error Description**
`test-4.0.2` gitian build fails with "file too big" in the custom authorities library. Adding `CMAKE_CXX_FLAGS=-Wa,-mbig-obj` doesn't help.

**Build Environment**
Details about the build environment, including the relevant required libraries. Much of this information can be found in the `CMakeFiles/CMakeOutput.log`. 
 - Host OS:             Ubuntu 18.04
 - Source Branch/Tag:   test-4.0.2
 - OpenSSL Version:     1.1.1d
 - Boost Version:       1.69.0
 - C++ Compiler:        x86_64-w64-mingw32-g++ (GCC) 7.3-win32 20180312

**Steps To Reproduce**
Checkout bitshares-gitian in the [changes_for_4.0.0 branch](https://github.com/bitshares/bitshares-gitian/pull/38), then

    run-gitian -b -O win test-4.0.2

**Console Logs (optional)**
```
[ 39%] Building CXX object libraries/protocol/CMakeFiles/graphene_protocol_custom_auths.dir/custom_authorities/list_1.cpp.obj
[ 39%] Building CXX object libraries/protocol/CMakeFiles/graphene_protocol_custom_auths.dir/custom_authorities/list_2.cpp.obj
/usr/bin/x86_64-w64-mingw32-as: CMakeFiles/graphene_protocol_custom_auths.dir/custom_authorities/list_1.cpp.obj: section .xdata$_ZN2fc8typelist4impl15dispatch_helperIZN8graphene8protocol14make_predicateINS4_12predicate_inENS4_5priceENS_14static_variantIJN5boost9container8flat_setIbSt4lessIbENSA_13new_allocatorIbEEEENSB_INS4_15public_key_typeESC_ISH_ENSE_ISH_EEEENSB_INS_6sha256ESC_ISL_ENSE_ISL_EEEENSB_IxSC_IxENSE_IxEEEENSB_INSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEESC_ISX_ENSE_ISX_EEEENSB_INS_14time_point_secESC_IS11_ENSE_IS11_EEEENSB_INS3_2db9object_idILh1ELh2EEESC_IS17_ENSE_IS17_EEEENSB_INS16_ILh1ELh3EEESC_IS1B_ENSE_IS1B_EEEENSB_INS16_ILh1ELh4EEESC_IS1F_ENSE_IS1F_EEEENSB_INS16_ILh1ELh5EEESC_IS1J_ENSE_IS1J_EEEENSB_INS16_ILh1ELh6EEESC_IS1N_ENSE_IS1N_EEEENSB_INS16_ILh1ELh7EEESC_IS1R_ENSE_IS1R_EEEENSB_INS16_ILh1ELh8EEESC_IS1V_ENSE_IS1V_EEEENSB_INS16_ILh1ELh9EEESC_IS1Z_ENSE_IS1Z_EEEENSB_INS16_ILh1ELh10EEESC_IS23_ENSE_IS23_EEEENSB_INS16_ILh1ELh12EEESC_IS27_ENSE_IS27_EEEENSB_INS16_ILh1ELh13EEESC_IS2B_ENSE_IS2B_EEEENSB_INS16_ILh1ELh14EEESC_IS2F_ENSE_IS2F_EEEENSB_INS16_ILh1ELh15EEESC_IS2J_ENSE_IS2J_EEEEEEEEESt8functionIFNS4_16predicate_resultERKT0_EET1_EUlT_E_S2O_IFS2P_RKS7_EENS0_7runtime7wrapperIS2A_EEEES2Q_RS2W_: string table overflow at offset 10000889
/tmp/cczGRLKO.s: Assembler messages:
/tmp/cczGRLKO.s: Fatal error: can't close CMakeFiles/graphene_protocol_custom_auths.dir/custom_authorities/list_1.cpp.obj: File too big
make[3]: *** [libraries/protocol/CMakeFiles/graphene_protocol_custom_auths.dir/custom_authorities/list_1.cpp.obj] Error 1
make[3]: *** Waiting for unfinished jobs....
libraries/protocol/CMakeFiles/graphene_protocol_custom_auths.dir/build.make:88: recipe for target 'libraries/protocol/CMakeFiles/graphene_protocol_custom_auths.dir/custom_authorities/list_1.cpp.obj' failed
/usr/bin/x86_64-w64-mingw32-as: CMakeFiles/graphene_protocol_custom_auths.dir/custom_authorities/list_2.cpp.obj: section .pdata$_ZN2fc8typelist4impl15dispatch_helperIZN8graphene8protocol14make_predicateINS4_16predicate_not_inENS_8optionalINS4_15account_optionsEEENS_14static_variantIJN5boost9container8flat_setIbSt4lessIbENSC_13new_allocatorIbEEEENSD_INS4_15public_key_typeESE_ISJ_ENSG_ISJ_EEEENSD_INS_6sha256ESE_ISN_ENSG_ISN_EEEENSD_IxSE_IxENSG_IxEEEENSD_INSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEESE_ISZ_ENSG_ISZ_EEEENSD_INS_14time_point_secESE_IS13_ENSG_IS13_EEEENSD_INS3_2db9object_idILh1ELh2EEESE_IS19_ENSG_IS19_EEEENSD_INS18_ILh1ELh3EEESE_IS1D_ENSG_IS1D_EEEENSD_INS18_ILh1ELh4EEESE_IS1H_ENSG_IS1H_EEEENSD_INS18_ILh1ELh5EEESE_IS1L_ENSG_IS1L_EEEENSD_INS18_ILh1ELh6EEESE_IS1P_ENSG_IS1P_EEEENSD_INS18_ILh1ELh7EEESE_IS1T_ENSG_IS1T_EEEENSD_INS18_ILh1ELh8EEESE_IS1X_ENSG_IS1X_EEEENSD_INS18_ILh1ELh9EEESE_IS21_ENSG_IS21_EEEENSD_INS18_ILh1ELh10EEESE_IS25_ENSG_IS25_EEEENSD_INS18_ILh1ELh12EEESE_IS29_ENSG_IS29_EEEENSD_INS18_ILh1ELh13EEESE_IS2D_ENSG_IS2D_EEEENSD_INS18_ILh1ELh14EEESE_IS2H_ENSG_IS2H_EEEENSD_INS18_ILh1ELh15EEESE_IS2L_ENSG_IS2L_EEEEEEEEESt8functionIFNS4_16predicate_resultERKT0_EET1_EUlT_E_S2Q_IFS2R_RKS9_EENS0_7runtime7wrapperIS16_EEEES2S_RS2Y_: string table overflow at offset 10000764
/tmp/ccdXOn08.s: Assembler messages:
/tmp/ccdXOn08.s: Fatal error: can't close CMakeFiles/graphene_protocol_custom_auths.dir/custom_authorities/list_2.cpp.obj: File too big
libraries/protocol/CMakeFiles/graphene_protocol_custom_auths.dir/build.make:113: recipe for target 'libraries/protocol/CMakeFiles/graphene_protocol_custom_auths.dir/custom_authorities/list_2.cpp.obj' failed
make[3]: *** [libraries/protocol/CMakeFiles/graphene_protocol_custom_auths.dir/custom_authorities/list_2.cpp.obj] Error 1
CMakeFiles/Makefile2:1714: recipe for target 'libraries/protocol/CMakeFiles/graphene_protocol_custom_auths.dir/all' failed
```

## CORE TEAM TASK LIST
- [ ] Evaluate `Build Error`
- [ ] Provide build guidance
- [ ] <OR> Create `Bug Report`

**Comment by pmconrad** [comment]
Related: #2003 #2028

**Comment by pmconrad** [comment]
Resolved by #2069
